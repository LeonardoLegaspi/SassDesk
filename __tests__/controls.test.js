var sass = require('node-sass');

describe('-----CONTROL TEST', ()=>{
    test('Control snapshot', async () => {
  
        var result = sass.renderSync({
          data: `
          @import './2_base/controls.scss';
          `,
          outputStyle: 'nested'
        });
        
        expect(result.css.toString()).toMatchSnapshot();
      });
  })