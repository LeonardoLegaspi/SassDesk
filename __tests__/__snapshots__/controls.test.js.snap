// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-----CONTROL TEST Control snapshot 1`] = `
"/*
FIXME Is this useful? or limits to buttons with colors on Variables?
*/
"
`;
